File: java/996A.java
// 996 A. Hit the Lottery

import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        int[] notes = {100, 20, 10, 5, 1};
        int count = 0;

        for (int note : notes) {
            count += n / note;
            n %= note;
        }

        System.out.println(count);
    }
}

File: cpp/996A.cpp
// 996 A. Hit the Lottery

#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;

    int notes[] = {100, 20, 10, 5, 1};
    int count = 0;
    for (int i = 0; i < 5; i++) {
        count += n / notes[i];
        n %= notes[i];
    }
    cout << count << endl;
    return 0;
}
